### ISSUE TITLE
owncloud integration missing

### ISSUE THREAD

**Issue by igormcoelho** [SimpleMobileTools/Simple-Calendar#631]
Hi, this is a nice software, but I really miss owncloud caldav integration. Do you have plans for that in the roadmap ?

**Comment by tibbi** [comment]
not sure what you mean. If you toggle "CalDAV sync" in the app settings, dont you see some calendars for syncing?

**Comment by igormcoelho** [comment]
Sorry, my bad! I just discovered I needed another app to make "the bridge" (Caldav Sync Free Beta), logged in to OwnCloud, then it displayed on SimpleCalendar. Perhaps it's good to add this information somewhere, for newbies like me, but I don't know where exactly :) congratulations!

**Comment by igormcoelho** [comment]
Guess I just found some weird bug here... My owncloud CalDav is connected, and the events display on the app (but not on the widget). I tried to personalize the colors of the calendar events, then I got this weird situation. 
![whatsapp image 2018-09-22 at 08 23 25](https://user-images.githubusercontent.com/290989/45916705-f1e06700-be40-11e8-9d07-566381a899a1.jpeg)

If I click Holidays ("Feriados"), I get a coloring option (OK):
![whatsapp image 2018-09-22 at 08 21 28](https://user-images.githubusercontent.com/290989/45916711-12a8bc80-be41-11e8-8574-8ae8933fdc05.jpeg)

If I click my google account CalDav, I get another select color option (which is strange):
![whatsapp image 2018-09-22 at 08 23 25 1](https://user-images.githubusercontent.com/290989/45916723-30762180-be41-11e8-97be-9a2f638853b3.jpeg)

And if I click my owncloud CalDav account, my screen is bugged:
![whatsapp image 2018-09-22 at 08 23 25 2](https://user-images.githubusercontent.com/290989/45916725-47b50f00-be41-11e8-9913-17d25bb7d2cd.jpeg)

hahah very strange... any advices? :)
